explainer.md

```diff
@@ -301,27 +296,17 @@ Badge.clear();
 ```
 
 ## UX treatment
-Badges should appear as a user agent defined overlay on top of the app's icon, about a quarter of the size, and in one of the four corners. If the exact representation of a badge is not supported (e.g., a 2-digit number while only single characters are allowed, or a character where only numbers are allowed) the user agent should make a best effort attempt to map the unsupported data onto supported data. This may involve:
+Badges may appear in any place that the user agent deems appropriate. In general, these places should be obviously related to the pages being badged, so users understand what the status is for. Appropriate places could include:
+- Tab favicons.
+- Bookmark icons.
+- [OS Specific Contexts](#OS-Specific-Contexts) for [Installed Web Applications](https://www.w3.org/TR/appmanifest/#installable-web-applications).
```

Comment:
I think this should be more concrete (about showing it over the app icon), like the previous text.

The difference between an explainer and a spec is that the spec has to play this weird game of pretending that it doesn't know anything about the real world and talks about everything in vague totally general terms. In the explainer, that is the place to be very concrete about how you actually intend this to be used.